packit/cli/__init__.py

```diff
@@ -1,2 +1,15 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
+
+import click
+
+from packit.cli.config import config  # Import the new config group
+
+
+@click.group()
+def cli():
+    pass
+
+
+if __name__ == "__main__":
+    cli()
```

Comment:
I would say this is not needed as we are initialising it [here](https://github.com/packit/packit/blob/69afa2b7e5e8e351c548b5c4f97b42c322658579/packit/cli/packit_base.py#L87)